packages/widgets/README.md

```diff
@@ -29,3 +36,16 @@ OAUTH_CLIENT_SECRET=...
 ```
 
 If you want to try the "Sign-in with HF" feature in a Space, you can just duplicate https://huggingface.co/spaces/huggingfacejs/inference-widgets, it should work out of the box thanks to the metadata in the `README.md` file.
+
+## Testing for moon (for huggingface admins)
```

Comment:
nit: I'd have done it the other way around (add a section in moon) or just give about "how to develop within your project"

Reply:
indeed